decidim-initiatives/app/queries/decidim/initiatives/admin/manageable_initiatives.rb

```diff
@@ -48,12 +48,12 @@ def query
 
           organization.available_locales.each_with_index do |loc, index|
             base = if index.zero?
-                     base.where("title->>? ilike ?", loc, "#{q}%")
-                         .or(Initiative.where("description->>? ilike ?", loc, "#{q}%"))
+                     base.where("title->>? ilike ?", loc, "%#{q}%")
```

Comment:
Hey, what does this syntax `title->>? ilike ?` mean? is is standard postgres?

Reply:
Yes, it's standard Postgres!

This get's translated to `title->>ca ilike %search_query%` for exqmple, the first `?` is just replaced for each available locale.

Reply:
OMG! this is SQL for json columns, great! :+1: